cloudbuild.yaml

```diff
@@ -1,14 +1,51 @@
 steps:
-  - id: deploy
+  - id: deploy-python
     name: "python:3.7-buster"
     entrypoint: "bash"
     args:
       - "-c"
       - |
+        python setup.py bdist_wheel
         if [ "$TAG_NAME" ] ; then
           pip install twine keyrings.google-artifactregistry-auth
-          python setup.py bdist_wheel
           twine upload --repository-url https://europe-west4-python.pkg.dev/dev-vml-cm/vml-apis-python/ dist/*
         fi
     env:
       - "TAG_NAME=${TAG_NAME}"
+
+  - id: deploy-csharp
+    name: "bitnami/dotnet-sdk"
+    entrypoint: "bash"
+    args:
+      - "-c"
+      - |
+        cp vmlapis.csproj gen/csharp
+        cd gen/csharp
+        if [ "$TAG_NAME"  ] ; then
+          dotnet pack -o . -c Release
+          dotnet nuget push vmlapis.$TAG_NAME.nupkg --source https://api.nuget.org/v3/index.json --api-key $$NUGET_API_KEY
+        else 
+          export TAG_NAME=0.0.0
+          dotnet pack -o . -c Release
+        fi
```

Comment:
What are these here for? I see they are in Java as well... it this something that's needed for the build of the package (even though it's not being released)?

Reply:
yes, version is required in project [here](https://github.com/e-conomic/vmlapis/blob/bf4e7bfc0cebd68e16b563b9ef16d2d0d61a3ce4/vmlapis.csproj#L7) and [here](https://github.com/e-conomic/vmlapis/blob/bf4e7bfc0cebd68e16b563b9ef16d2d0d61a3ce4/java-package/pom.xml#L9) to build a package regardless of if it's going to be released or not